core/src/main/java/nl/nn/adapterframework/http/HttpSenderBase.java

```diff
@@ -796,33 +814,42 @@ public void setUserName(String username) {
 		setUsername(username);
 	}
 
-	@IbisDoc({"22", "password used in authentication to host", " "})
+	@IbisDoc({"password used in authentication to host", " "})
 	public void setPassword(String string) {
 		password = string;
 	}
 
-	@IbisDoc({"23", "domain used in authentication to host", " "})
+	@IbisDoc({"domain used in authentication to host", " "})
 	public void setAuthDomain(String string) {
 		authDomain = string;
 	}
 
+	/** endpoint to obtain OAuth accessToken using ClientCredentials grant. ClientID/ClientSecret are taken from authAlias, username and password. */
+	public void setTokenEndpoint(String string) {
+		tokenEndpoint = string;
+	}
+	/** comma separated list of scope items, only used when tokenEndpoint is specified */
+	public void setScope(String string) {
+		scope = string;
+	}
```

Comment:
dit is verwarrend, scope moet het tokenEndpoint + url zijn

Reply:
Nee, scope is de scope die gebruikt gaat worden bij de aanvraag van het token

Reply:
wat is dan een voorbeeld van een scope?